### DISCUSSION TITLE
Where do I look at the RX TX communication pins for different chips

### DISCUSSION THREAD

**Discussion by ahonxxx** [Koenkk/Z-Stack-firmware#487] [Q&A] [answered]
I'm currently using the <CC2652P(RFSTAR RF-BM-2652P2)>module,  Rx→DIO12   Tx→DIO13
Other chips, how to view
Thank you

**Comment by Hedda** [comment, marked as answer]
Most are listed in this README -> https://github.com/Koenkk/Z-Stack-firmware/blob/master/coordinator/Z-Stack_3.x.0/bin/README.md

Can also look in firmware.patch -> https://github.com/Koenkk/Z-Stack-firmware/blob/master/coordinator/Z-Stack_3.x.0/firmware.patch

Otherwise, you need to look at the SoC chip or radio module specification/documentation from the manufacturer, e.g. from RF-Star:

https://www.szrfstar.com/upload/file/1598420209.pdf
